src/config/config.js

```diff
@@ -284,6 +272,24 @@ export const config = convict({
       format: Boolean,
       default: false,
       env: 'NOTIFY_EMAIL_DOWNLOAD_CONFIRMATION'
+    },
+    caseDeliveryTemplateId: /** @type {SchemaObj<string | null>} */ {
```

Comment:
would it not make sense to have a `caseDeliveryObject` with the 2 props?

Reply:
I was expecting that comment tbh, I wasn't sure whether it was worth it or not, but I'll create two, one for caseDelivery and one for applicant

Reply:
done